### PR TITLE
Fix for running in windows environments

### PR DESCRIPTION
**Pull request by Salivala**
directory scanning only taking into account linux/unix dir naming

### CODE REVIEW

File: src/cogs/mirror/mirror.py [hunk 1]
```diff
@@ -6,33 +6,35 @@
...
+        if 'mirror_cache' not in self.cache.keys():
+            self.cache['mirror_cache']: dict = {ctx.message.channel.id: ctx.message.channel_mentions[0].id}
         else:
-            self.bot.cache['mirror_cache'][ctx.message.channel.id] = ctx.message.channel_mentions[0].id
+            self.cache['mirror_cache'][ctx.message.channel.id] = ctx.message.channel_mentions[0].id
+
 
 def setup(bot):
-    bot.add_cog(Mirror(bot))
+    bot.add_cog(Mirror(bot, bot.cache))
```
Comment (holdank) [inline src/cogs/mirror/mirror.py:40]: it doesn't complain when load_extension is used?  i thought i had tried this and it gave me an argument error

Comment (holdank) [reply src/cogs/mirror/mirror.py:40]: wait why do you need to pass it here if we already have it on `bot`?

**Review by holdank — APPROVED** [review]

File: src/cogs/mirror/mirror.py (hunk 1 above)
Comment (pblca) [reply src/cogs/mirror/mirror.py:40]: Seconded, this should just be derived from the bot

**Review by pblca — CHANGES_REQUESTED** [review]

File: src/cogs/mirror/mirror.py (hunk 1 above)
Comment (Salivala) [reply src/cogs/mirror/mirror.py]: moved it back into class

**Pull request closed without merging**